pkg/executor/scraper/minio_loader.go

```diff
@@ -0,0 +1,64 @@
+package scraper
+
+import (
+	"context"
+
+	"github.com/kubeshop/testkube/pkg/api/v1/testkube"
+
+	"github.com/pkg/errors"
+
+	"github.com/kubeshop/testkube/pkg/storage/minio"
+	"github.com/kubeshop/testkube/pkg/utils"
+)
+
+type MinIOLoader struct {
+	Endpoint, AccessKeyID, SecretAccessKey, Location, Token, Bucket string
+	Ssl                                                             bool
+	client                                                          *minio.Client
+}
+
+func NewMinIOLoader(endpoint, accessKeyID, secretAccessKey, location, token, bucket string, ssl bool) (*MinIOLoader, error) {
+	l := &MinIOLoader{
+		Endpoint:        endpoint,
+		AccessKeyID:     accessKeyID,
+		SecretAccessKey: secretAccessKey,
+		Location:        location,
+		Token:           token,
+		Bucket:          bucket,
+		Ssl:             ssl,
+	}
+
+	client := minio.NewClient(l.Endpoint, l.AccessKeyID, l.SecretAccessKey, l.Location, l.Token, l.Bucket, l.Ssl)
+	err := client.Connect()
+	if err != nil {
+		return nil, errors.Errorf("error occured creating minio client: %v", err)
+	}
+	l.client = client
+
+	return l, nil
+}
+
+func (l *MinIOLoader) Load(ctx context.Context, object *Object, meta map[string]any) error {
```

Comment:
Looks like `Load` just Store/Save/Uploads file with `client.SaveFileDirect(` so Load is confusing